tests/lib/linter/timing.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const { getListSize } = require("../../../lib/linter/timing");
+const assert = require("chai").assert;
+
+describe("timing", () => {
+    describe("getListSize()", () => {
+        it("returns minimum list size with small environment variable value", () => {
+            process.env.TIMING = true;
+            assert.strictEqual(getListSize(), 10);
+
+            process.env.TIMING = "foo";
+            assert.strictEqual(getListSize(), 10);
+
+            process.env.TIMING = 0;
+            assert.strictEqual(getListSize(), 10);
+
+            process.env.TIMING = 1;
+            assert.strictEqual(getListSize(), 10);
+
+            process.env.TIMING = 5;
+            assert.strictEqual(getListSize(), 10);
+
+            process.env.TIMING = 10;
+            assert.strictEqual(getListSize(), 10);
+        });
+
+        it("returns longer list size with larger environment variable value", () => {
+            process.env.TIMING = 11;
+            assert.strictEqual(getListSize(), 11);
+
+            process.env.TIMING = 100;
+            assert.strictEqual(getListSize(), 100);
+        });
+
+        it("returns maximum list size with environment variable value of 'all'", () => {
+            process.env.TIMING = "all";
```

Comment:
Should we restore `process.env` to prevent these changes from leaking to other unit tests? I guess it doesn't affect them at the moment, but still looks like the right thing to do.

Reply:
Good point. I added an `after` hook to reset the value.